model/blog.js

```diff
@@ -11,7 +11,8 @@ const BlogSchema = new mongoose.Schema({
   },
   previewImg: {
     type: String,
-    required: false
+    required: false,
+    default:'/assets/blog_default_img.jpeg'
```

Comment:
You need to add a space character after 'default':
```js
default: '/assets/blog_default_img.jpeg'
```
Running `npm run fix` will format your code automatically.